### DISCUSSION TITLE
Dell PN557W Pen: Cannot authenticate and/or hold a connection/pair

### DISCUSSION THREAD

**Discussion by sammypanda** [blueman-project/blueman#2452] [Q&A]
blueman: 2.2.4-1
BlueZ: 5.64-0ubuntu1pop1\~1674313994\~22.04~579884f
Distribution: Pop! OS
Desktop environment: Gnome

When trying to connect to the active pen, it usually throws 'AuthenticationFailed' with bluetoothctl:
> [CHG] Device 00:6F:F2:E0:DD:22 Connected: yes
Failed to pair: org.bluez.Error.AuthenticationFailed
[CHG] Device 00:6F:F2:E0:DD:22 Connected: no

But using Blueman it (*appears*) to get further, before failing in a way i don't understand.

Additionally, the prompt to pair never appears pair is selected first (the behavior seems the same as using bluetoothctl). It does appears if it's connected to, in which case it remains connected for a short period before disconnecting again. (Battery reports as 99%)

[Bluetooth Log 1](https://pastebin.com/kt4eMMdM)
[Blueman Log 2](https://pastebin.com/YEeYBAni)

**Comment by cschramm** [comment]
Not sure what difference blueman makes there, but in any case, I'd try more recent BlueZ / Linux / firmware. :crossed_fingers:
